plugins/get-all.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: get-all
+spec:
+  version: "v0.0.1"
+  platforms:
+    - uri: https://github.com/corneliusweig/ketall/releases/download/v0.0.1/bundle.tar.gz
+      sha256: 3ec69406ad0b912c4817c37a07a4b3e7866c46dbfaa6d5b3a2db43004374c43f
+      bin: kubectl-get-all
+      files:
+        - from: ./ketall-linux-amd64
+          to: kubectl-get-all
+      selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+    - uri: https://github.com/corneliusweig/ketall/releases/download/v0.0.1/bundle.tar.gz
+      sha256: 3ec69406ad0b912c4817c37a07a4b3e7866c46dbfaa6d5b3a2db43004374c43f
+      bin: kubectl-get-all
+      files:
+        - from: ./ketall-darwin-amd64
+          to: kubectl-get-all
+      selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+    - uri: https://github.com/corneliusweig/ketall/releases/download/v0.0.1/bundle.tar.gz
+      sha256: 3ec69406ad0b912c4817c37a07a4b3e7866c46dbfaa6d5b3a2db43004374c43f
+      bin: kubectl-get-all.exe
+      files:
+        - from: ./ketall-windows-amd64
+          to: kubectl-get-all.exe
+      selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+  shortDescription: Like 'kubectl get all', but get _really_ all resources
+  caveats: This plugin needs cluster admin rights, because it retrieves everything
```

Comment:
if you have usage string, or link to docs, put them in caveats (we show it post-install).

Reply:
done :)